jb/itch5/compute_book_depth.hpp

```diff
@@ -0,0 +1,50 @@
+#ifndef jb_itch5_compute_book_depth_hpp
+#define jb_itch5_compute_book_depth_hpp
+
+#include <jb/itch5/compute_base.hpp>
+
+namespace jb {
+namespace itch5 {
+
+/** 
+ * An implementation derivated fomr compute_base to callback the book depth.
+ */
+
+class compute_book_depth : public compute_base {
+ public:
+  //@{
+  /**
+   * @name Type traits
+   */
+  /// Callbacks
+  typedef std::function<void(time_point,
```

Comment:
formatting:  maybe we should consider using clang-fmt so we can turn this into a mechanical problem.